src/engine/server/server.cpp

```diff
@@ -854,6 +852,10 @@ void CServer::ProcessClientPacket(CNetChunk *pPacket)
 					return;
 				}
 
+				const char *pModAPI = Unpacker.GetString(CUnpacker::SANITIZE_CC);
+				if(str_comp(pModAPI, GameServer()->NetVersion()) == 0)
```

Comment:
You can check if the Unpacker return an error to be sure that you have read something. If you don't read something, you don't have to compare the NetVersion